### DISCUSSION TITLE
more objects than one in alternatives array?

### DISCUSSION THREAD

**Discussion by scoutaroom** [deepgram/community#552] [General help] [answered]
Hey all,
In which cases in response we will get more objects than one in alternatives array?
Thanks

**Comment by team-deepgram** [comment]
Thanks for asking your question about Deepgram! If you didn't already include it in your post, please be sure to add as much detail as possible so we can assist you efficiently, such as:
- The `request_id` if you have a question about your requests or transcription responses.
- The features you used or the full `api.deepgram.com` URL you sent your request to, including parameters.
- Any code snippets you can share.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @scoutaroom, the alternatives are a vestige of a prior feature where you could request several alternative transcripts based on probabilities. You can assume there will be only one alternative in your response.
